specification/columns/chargecategory.md

```diff
@@ -1,9 +1,6 @@
 # Charge Category
```

Comment:
_I am unable to directly add a comment to the **ChargeCategory - Description**, so, here it is:_ 
The Description requires some adjustments. It should closely resemble the introductory sentence in the column definition.

Reply:
Hey Irina, i think i got this resolved in the manual merge.